index.html

```diff
@@ -8,27 +8,50 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-        <header id="header">
+        <header class="header header_desktop">
             <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
            
             <nav><img class ="ic_menu" src="images/ic_menu.png"/></nav>
 
         </header>
+       
+        <ul class="menu">
+            <li class="links"><a href="#hello">Hello</a></li>
+            <li class="links"><a href="#portfolio">Portfolio</a></li>
+            <li class="links"><a href="#about">About</a></li>
+            <li class="links"><a href="#contact">Contact</a></li>
+         </ul>
+         
+      
+       
 
-        <section id="headline">
+        <section class="headline">
+            <div class="mb-headline">
             <h1 class="profileName">Harshita Jain</h1>
             
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me. </p>
             <button class="startbtn">Start collaboration</button>
```

Comment:
- [ ] Kindly increase the `font-size` of the text in this button and change the `font-family` to `Poppins` to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200124273-a3f92e17-2762-4554-886b-eeb76c330aa9.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200124298-25ed790b-6665-447b-a44a-05cee1110cef.png)

Reply:
Fixed.